docs/ContentTypeParser.md

```diff
@@ -28,6 +28,9 @@ fastify.addContentTypeParser('application/jsoff', async function (request, paylo
 
   return res
 })
+
+// Can use default JSON/Text parser for different content Types
+fastify.addContentTypeParser('text/json', { parseAs: 'string' }, fastify.getDefaultJsonParser('ignore', 'ignore')
```

Comment:
We need to document the new methods in https://github.com/fastify/fastify/blob/master/docs/Server.md